_posts/blog/2022-04-26-march-2022-community-feedback.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+categories: 
+  - blog
+title: "March 2022 Community Feedback Session"
+description: "On March 15, 2022, Chi Hack Night held a hybrid community feedback session
+to assess how our community is feeling about the current phase of the pandemic and what people
+are looking for from Chi Hack Night in this moment."
+date: 2022-04-26
+image: /images/blog/2022-04-26-march-2022-community-feedback/20220315_community_feedback_miro_board.png
+author: Laurie Merrell
+author_url: https://twitter.com/laurie_merrell
+author_image: /images/people/board/laurie_merrell.jpg
+author_bio: "Board Member, Chi Hack Night"
+author2: 
+author2_url:
+author2_bio:
+published: true
+featured: false
```

Comment:
to have the blog post show up on the home page, lets set `featured: true`

Reply:
we should also make the two currently featured blog posts no longer featured as they're pretty old now, so this one will be the only one featured on the home page

Reply:
Thanks Derek, resolved in fe048c934e78f9a5c62376d198ee161d4d406162 and 9ed131ed1ee3d756e1c406751356d53d84e0d2bf